docker-compose.yml

```
@@ -0,0 +1,10 @@
+services:
+  web:
+    build:
+      context: .
+      args:
+        version: v2024.6.12
+    container_name: harmony-container
+    image: harmony
```

Tagging the image with `$DENO_DEPLOYMENT_ID` might make sense? (I don't have much Deno experience, so not sure if this variable is ideal for versioning.)
```suggestion
    image: harmony:$DENO_DEPLOYMENT_ID
```